nss-day-cohort-39/hello-world-the-globetrotters
# Code cleanup after review

### Changes
- Removed obsolete code and commented out code.
- Removed unused javascript and html pages.

### Testing
- Make sure the page works.
- Inspect for any unused or obsolete elements that can be removed.
